File: 250811/특정 수 맞히기/catching-specific-number.c
#include <stdio.h>

int main() {
    int user_input;

    while(1) {
        scanf("%d", &user_input);

        if(user_input==25) {
            printf("Good\n");
            break;
        }

        if(user_input<25) {
            printf("Higher\n");
        }
        
        if(user_input>25) {
            printf("Lower\n");
        }
    }




    return 0;
}

File: 250811/특정 수 맞히기/catching-specific-number.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while(true) {
            int user_input = sc.nextInt();

            if(user_input==25) {
                System.out.println("Good");
                break;
            }

            if(user_input<25) {
                System.out.println("Higher");
            }

            if(user_input>25) {
                System.out.println("Lower");
            }
        
        }
    }
}